template/INSTALL.md.jinja

```diff
@@ -20,7 +20,7 @@ Download from {{ repository }}/releases
 
 ### Installing with Pixi (recommended)
 
-We recommend using `pixi` to manage a Python environment with `{{ project_name }}` and all the required dependencies since it can reproduce exactly what we use for development.
+We recommend using `pixi` to manage a Python environment with `{{ project_slug }}` and all the required dependencies since it can reproduce exactly what we use for development.
```

Comment:
Should we use the `name` or `slug` here? A pypi search would be a match on the name used in `pyproject.toml`, is that right?

Reply:
Shoot, I think you are right, I'll revert this